### PR TITLE
fix: Remove deprecated call to React.PropTypes

### PR DESCRIPTION
**Pull request by robtarr**

### CODE REVIEW

File: src/eq.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import React, { Component, cloneElement } from 'react';
+import { Component, cloneElement } from 'react';
+import PropTypes from 'prop-types';
```
Comment (pfulton) [inline src/eq.js:2]: Do we also need to include `prop-types` as one of the dependencies in the `package.json`?

**Review by pfulton — CHANGES_REQUESTED** [review]
Just one question here, @robtarr.

File: src/eq.js (hunk 1 above)
Comment (robtarr) [reply src/eq.js:2]: fixed.

**Comment by pfulton** [comment]
Do we also need to bump the version number in the `package.json` to `0.0.4`?

**Pull request merged**